app/code/Magento/Inventory/Test/Acceptance/Test/AdminOutOfStockThresholdAppliedToSimpleConfigurationPageTest.xml

```diff
@@ -0,0 +1,158 @@
+            <waitForPageLoad time="10" stepKey="waitForPageLoad4"/>
+            <click selector="{{AdminEditStockSourcesSection.assignSources}}" stepKey="clickOnAssignSources1"/>
+            <actionGroup ref="searchAdminDataGridByKeyword" stepKey="searchByKeyword2">
+                <argument name="keyword" value="$$createSource1.source[name]$$"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.checkboxByValue($$createSource1.source[name]$$)}}" stepKey="clickOnTheCheckboxForSource2"/>
+            <click selector="{{AdminManageSourcesGridControls.done}}" stepKey="clickOnDone1"/>
+            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveStock1"/>
+
+            <amOnPage url="{{AdminProductIndexPage.url}}" stepKey="navigateToProductIndex1"/>
+        </before>
+        <after>
+            <deleteData createDataKey="simpleCategory1" stepKey="deleteCategory1"/>
+
+            <!-- Disable source as it can broke single source mode tests. -->
+            <amOnPage url="{{AdminManageSourcePage.url}}" stepKey="amOnTheSourcesListPage1"/>
+            <conditionalClick selector="{{AdminDataGridHeaderSection.clearFilters}}" dependentSelector="{{AdminDataGridHeaderSection.clearFilters}}" visible="true" stepKey="clickClearFilters1"/>
+            <actionGroup ref="AdminGridFilterSearchResultsByInput" stepKey="filterSearchResults1">
+                <argument name="selector" value="AdminManageSourcesGridFilterControls.code"/>
+                <argument name="value" value="_minimalSource.source_code"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.editByValue($$createSource1.source[source_code]$$)}}" stepKey="clickOnEdit1"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad5"/>
+            <click selector="{{AdminEditSourceGeneralSection.IsEnabledLabel}}" stepKey="clickIsEnabled1"/>
+            <actionGroup ref="AdminFormSaveAndClose" stepKey="saveAndClose1"/>
+
+            <!-- Assign Default Stock to Main Website -->
+            <amOnPage url="{{AdminManageStockPage.url}}" stepKey="amOnTheStocksListPage2"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad6"/>
+            <actionGroup ref="searchAdminDataGridByKeyword" stepKey="searchByKeyword3">
+                <argument name="keyword" value="_defaultStock.name"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.editByValue(_defaultStock.name)}}" stepKey="clickOnEditForStock2"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad7"/>
+            <selectOption selector="{{AdminEditStockSalesChannelsSection.websites}}" userInput="Main Website" stepKey="SelectWebsiteAsSalesChannel2"/>
+            <waitForPageLoad time="10" stepKey="waitForPageLoad8"/>
+            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveStock2"/>
+
+            <!-- Set out of stock threshold to use system value. -->
```

Comment:
Good idea. It will make the test faster.
But I have one question: 
If I set it like this `<magentoCLI command="config:set cataloginventory/item_options/min_qty 1" stepKey="setMinQty1"/>` it will insert in db value '1' and if I go to admin, check `Use system value` and save configuration it will remove the min_qty value from db.
Can I remove it too via `magentoCLI` ?